ehr/src/org/labkey/ehr/utils/TriggerScriptHelper.java

```
@@ -134,10 +134,12 @@ public class TriggerScriptHelper
      *  <li>departureStatus - center specific custom status for animal departures</li>
      */
     @NotNull
-    private static final Map<String, String> _centerCustomProps = new HashMap<>();
+    private static final Map<String, Object> _centerCustomProps = new HashMap<>();
 
     private static final Logger _log = LogHelper.getLogger(TriggerScriptHelper.class, "Server-side validation of EHR data insert/update/deletes");
 
+    private static double _bloodNearingOveragesThresholdDefaultValue = 4.0; //in mL
```

Is this being used somewhere? I'm not finding any references to it.

---

It used to be, but since we're throwing errors instead of relying on a default val it is no longer being used. I removed this